src/ui/authenticationInApp/ViewAuthenticationVerificationSessionByEmailNeeded.qml

```diff
@@ -58,7 +64,13 @@ VPNInAppAuthenticationBase {
 
         VPNCancelButton {
             anchors.horizontalCenter: parent.horizontalCenter
-            onClicked: VPN.cancelAuthentication()
+            onClicked: {
```

Comment:
Would be nice if we could consolidate these `if (isDeleteAccountAuth) { cancelAccountDeletion() }.. else { VPN.cancelAuthentication() }` somehow.